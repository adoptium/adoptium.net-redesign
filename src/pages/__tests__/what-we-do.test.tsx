import React from "react"
import { render } from "@testing-library/react"
import { describe, expect, it } from "vitest"
import { axe } from "vitest-axe"
import Index, { Head } from "../what-we-do"

describe("What We Do page", () => {
  it("renders correctly", () => {
    const { container } = render(<Index />)
    // eslint-disable-next-line
    const pageContent = container.querySelector("main")

    expect(pageContent).toMatchSnapshot()
  })

  it("head renders correctly", () => {
    const { container } = render(<Head />)
    // eslint-disable-next-line
    const title = container.querySelector("title")
    expect(title?.textContent).toEqual("What We Do | Adoptium")
  })

  it("has no accessibility violations", async () => {
    const { container } = render(<Index />)
    const results = await axe(container)
    // @ts-ignore
    expect(results).toHaveNoViolations()
  })
})
