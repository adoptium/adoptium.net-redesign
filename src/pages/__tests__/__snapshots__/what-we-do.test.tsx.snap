// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`What We Do page > renders correctly 1`] = `
<main>
  <div
    class="relative pt-40 pb-16 md:pb-36 md:pt-60 overflow-hidden"
  >
    <div
      class="w-full relative"
    >
      <div
        class="absolute z-[-1] w-[2396px] h-[1340px] left-[23px] md:left-[112px] top-[21px] md:top-[171px] bg-[#410170] shadow-[0_0_400px_rgba(65,1,112,1)] rounded-full blur-[400px]"
      />
      <div
        class="absolute z-[-1] w-[1487px] h-[893px] left-[120px] md:left-[676px] top-[120px] md:top-[395px] bg-[#B62987] shadow-[0_0_400px_rgba(182,41,135,1)] rounded-full blur-[400px]"
      />
      <div
        class="absolute z-[-1] w-[688px] h-[446px] left-[400px] md:left-[1131px] top-[618px] bg-[#FE8492] shadow-[0_0_400px_rgba(254,132,146,1)] rounded-full blur-[400px]"
      />
    </div>
    <div
      class="mx-auto max-w-[1048px] w-full px-6 lg:px-0 flex flex-col items-center justify-center"
    >
      <div
        class="self-stretch flex-col justify-center items-center gap-6 flex"
      >
        <div
          class="self-stretch  flex-col justify-center items-center gap-4 flex"
        >
          <div
            class="justify-start items-center gap-3 inline-flex"
          >
            <svg
              fill="none"
              height="14"
              viewBox="0 0 14 14"
              width="14"
              xmlns="http://www.w3.org/2000/svg"
            >
              <rect
                fill="#FF1464"
                height="8"
                rx="2"
                width="8"
                x="3"
                y="3"
              />
              <rect
                height="11"
                rx="3.5"
                stroke="#FF1464"
                stroke-opacity="0.25"
                stroke-width="3"
                width="11"
                x="1.5"
                y="1.5"
              />
            </svg>
            <div
              class="text-rose-600 text-base font-bold leading-normal"
            >
              What we do
            </div>
          </div>
          <div
            class="self-stretch text-center text-white   text-[56px] lg:text-[80px] leading-[114.286%] md:leading-[120%] font-semibold"
          >
            About Eclipse Adoptium®
          </div>
        </div>
        <div
          class="self-stretch text-center text-lightgrey text-[20px] font-normal leading-[140%] undefined"
        >
          <div
            class="text-center lg:text-justify flex flex-col ml-10 gap-3"
          >
            <p>
              Adoptium champions open-source Java solutions, empowering businesses with cutting-edge technology and collaborative innovation.
            </p>
            <p>
              Founded on the principles of community-driven development, Adoptium has become a cornerstone of the Java ecosystem, driving forward-thinking solutions and fostering a culture of shared knowledge.
            </p>
            <p>
              We achieve this through a set of Projects under the Adoptium PMC and a close working partnership with external projects, most notably OpenJDK for providing the Java SE runtime implementation.
            </p>
            <p>
              The AdoptOpenJDK project was established in 2017 following years of discussions about the general lack of an open and reproducible build and test system for OpenJDK source across multiple platforms. Since then it has grown to become a leading provider of high-quality OpenJDK-based binaries used by enterprises in embedded systems, desktops, traditional servers, modern cloud platforms, and large mainframes. The Eclipse Adoptium project is the continuation of the original AdoptOpenJDK mission.
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="bg-blue py-24 sm:py-28"
  >
    <div
      class="mx-auto max-w-3xl md:max-w-7xl w-full px-6 xl:px-0"
    >
      <div
        class="mx-auto max-w-4xl sm:text-center"
      >
        <h2
          class="text-center text-2xl md:text-5xl font-semibold text-white md:leading-[56px] leading-[44px] mb-4"
        >
          Discover all our projects
        </h2>
      </div>
      <div
        class="flex flex-col justify-center items-center"
      >
        <p
          class="mt-6 max-w-2xl mb-0 text-base leading-6 text-grey text-center md:text-justify md:[text-align-last:center]"
        >
          Eclipse Adoptium provides runtime binaries that are high performance, enterprise-caliber, cross-platform, open-source licensed, Java SE TCK-tested and compliant for general use across the Java ecosystem. Eclipse Adoptium also provides artifacts including open-source infrastructure as code, a comprehensive continuous integration build and test farm, and extensive quality assurance tests. These artifacts may be used by Eclipse Adoptium projects and other runtime technology builders to ensure the provision of secure, Java SE TCK-tested and compliant, production-ready runtimes.
        </p>
      </div>
      <div
        class="mt-16 mb-8 flex md:h-[30em] justify-around lg:justify-between flex-wrap lg:flex-nowrap gap-6 xl:gap-8"
      >
        <div
          class="max-w-xl w-full h-full"
        >
          <div
            class="bg-white/5 flex justify-between h-full flex-col p-8 xl:p-12 rounded-3xl border-2 border-white/50"
          >
            <img
              alt="Eclipse Temurin™ logo"
              class="w-[140px]"
              src="/images/projects/temurin-project.svg"
            />
            <div
              class="flex-col justify-center items-start gap-2"
            >
              <div
                class="justify-start items-center gap-3 inline-flex pt-4"
              >
                <svg
                  fill="none"
                  height="14"
                  viewBox="0 0 14 14"
                  width="14"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <rect
                    fill="#FF1464"
                    height="8"
                    rx="2"
                    width="8"
                    x="3"
                    y="3"
                  />
                  <rect
                    height="11"
                    rx="3.5"
                    stroke="#FF1464"
                    stroke-opacity="0.25"
                    stroke-width="3"
                    width="11"
                    x="1.5"
                    y="1.5"
                  />
                </svg>
                <p
                  class="text-rose-600 mb-0 text-base leading-6 font-bold"
                >
                  Runtimes and Development Kits
                </p>
              </div>
              <h2
                class="text-white text-xl md:text-3xl leading-8 md:leading-10 font-semibold mt-2"
              >
                Eclipse Temurin™
              </h2>
              <a
                href="/en/temurin"
              >
                <button
                  class="bg-transparent border-2 mt-8 sm:mt-10 border-pink-500/0 text-whit text-base leading-6 font-bold w-[148px] h-[48px] rounded-2xl gradient-border"
                >
                  Learn More
                </button>
              </a>
            </div>
          </div>
        </div>
        <div
          class="max-w-xl w-full h-full"
        >
          <div
            class="bg-white/5 flex justify-between h-full flex-col p-8 xl:p-12 rounded-3xl border-2 border-white/50"
          >
            <img
              alt="Eclipse AQAvit logo"
              class="w-[140px]"
              src="/images/projects/aqavit-project.svg"
            />
            <div
              class="flex-col justify-center items-start gap-2"
            >
              <div
                class="justify-start items-center gap-3 inline-flex pt-4"
              >
                <svg
                  fill="none"
                  height="14"
                  viewBox="0 0 14 14"
                  width="14"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <rect
                    fill="#FF1464"
                    height="8"
                    rx="2"
                    width="8"
                    x="3"
                    y="3"
                  />
                  <rect
                    height="11"
                    rx="3.5"
                    stroke="#FF1464"
                    stroke-opacity="0.25"
                    stroke-width="3"
                    width="11"
                    x="1.5"
                    y="1.5"
                  />
                </svg>
                <p
                  class="text-rose-600 mb-0 text-base leading-6 font-bold"
                >
                  Quality Assurance
                </p>
              </div>
              <h2
                class="text-white text-xl md:text-3xl leading-8 md:leading-10 font-semibold mt-2"
              >
                Eclipse AQAvit
              </h2>
              <a
                href="/en/aqavit"
              >
                <button
                  class="bg-transparent border-2 mt-8 sm:mt-10 border-pink-500/0 text-whit text-base leading-6 font-bold w-[148px] h-[48px] rounded-2xl gradient-border"
                >
                  Learn More
                </button>
              </a>
            </div>
          </div>
        </div>
        <div
          class="max-w-xl w-full h-full"
        >
          <div
            class="bg-white/5 flex justify-between h-full flex-col p-8 xl:p-12 rounded-3xl border-2 border-white/50"
          >
            <img
              alt="Eclipse Mission Control logo"
              class="w-[140px]"
              src="/images/projects/mc-project.svg"
            />
            <div
              class="flex-col justify-center items-start gap-2"
            >
              <div
                class="justify-start items-center gap-3 inline-flex pt-4"
              >
                <svg
                  fill="none"
                  height="14"
                  viewBox="0 0 14 14"
                  width="14"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <rect
                    fill="#FF1464"
                    height="8"
                    rx="2"
                    width="8"
                    x="3"
                    y="3"
                  />
                  <rect
                    height="11"
                    rx="3.5"
                    stroke="#FF1464"
                    stroke-opacity="0.25"
                    stroke-width="3"
                    width="11"
                    x="1.5"
                    y="1.5"
                  />
                </svg>
                <p
                  class="text-rose-600 mb-0 text-base leading-6 font-bold"
                >
                  JDK Mission Control
                </p>
              </div>
              <h2
                class="text-white text-xl md:text-3xl leading-8 md:leading-10 font-semibold mt-2"
              >
                Eclipse Mission Control
              </h2>
              <a
                href="/en/jmc"
              >
                <button
                  class="bg-transparent border-2 mt-8 sm:mt-10 border-pink-500/0 text-whit text-base leading-6 font-bold w-[148px] h-[48px] rounded-2xl gradient-border"
                >
                  Learn More
                </button>
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <section
    class="py-16 md:py-32 mx-auto bg-[#14003C] max-w-[1264px] w-full lg:px-2   xl:px-0 px-8"
  >
    <div
      class="text-center"
    >
      <h2
        class="text-4xl leading-[122%] md:text-5xl md:[116%] text-white font-semibold"
      >
        Ways to support us
      </h2>
      <p
        class="text-grey mt-6 font-normal text-base"
      >
        If you value Eclipse technologies, please consider becoming a sponsor through the Eclipse Foundation. Contributions from users like you help fund the operations of the Adoptium working group. All money contributed to the Eclipse Foundation will be used to support the Eclipse community through the Adoptium working group
      </p>
    </div>
    <div
      class="mt-16  gap-4 md:gap-8  flex flex-wrap justify-center items-center "
    >
      <div
        class=""
      >
        <div
          class="p-10 flex flex-col max-w-[400px] newscard-2 h-[440px] md:h-[480px] bg-[#200E46]"
        >
          <div
            class="grow"
          >
            <h3
              class="text-white text-[30px] font-semibold  whitespace-nowrap  md:whitespace-normal leading-[120%] md:text-[40px]"
            >
              How to contribute
            </h3>
            <p
              class="mt-6 text-xl text-grey font-normal"
            >
              Contribute to our projects with technical and non-technical development
            </p>
          </div>
          <a
            href="/en/contributing/"
          >
            <button
              class="bg-transparent   border-2 border-pink-500/0 text-white text-base leading-6 font-normal w-[180px] h-[48px] rounded-[80px] gradient-border "
            >
              How to contribute
            </button>
          </a>
        </div>
      </div>
      <div
        class=""
      >
        <div
          class="p-10 flex flex-col max-w-[400px] newscard-2 h-[440px] md:h-[480px] bg-[#200E46]"
        >
          <div
            class="grow"
          >
            <h3
              class="text-white text-[30px] font-semibold  whitespace-nowrap  md:whitespace-normal leading-[120%] md:text-[40px]"
            >
              Become a sustainer
            </h3>
            <p
              class="mt-6 text-xl text-grey font-normal"
            >
              Join us in strengthening the future of Eclipse Temurin
            </p>
          </div>
          <a
            href="/en/sustainers"
          >
            <button
              class="bg-transparent   border-2 border-pink-500/0 text-white text-base leading-6 font-normal w-[180px] h-[48px] rounded-[80px] gradient-border "
            >
              Become a sustainer
            </button>
          </a>
        </div>
      </div>
      <div
        class=""
      >
        <div
          class="p-10 flex flex-col max-w-[400px] newscard-2 h-[440px] md:h-[480px] bg-[#200E46]"
        >
          <div
            class="grow"
          >
            <h3
              class="text-white text-[30px] font-semibold  whitespace-nowrap  md:whitespace-normal leading-[120%] md:text-[40px]"
            >
              Become a member
            </h3>
            <p
              class="mt-6 text-xl text-grey font-normal"
            >
              Join us shaping the future of Eclipse Adoptium
            </p>
          </div>
          <a
            href="/en/join"
          >
            <button
              class="bg-transparent   border-2 border-pink-500/0 text-white text-base leading-6 font-normal w-[180px] h-[48px] rounded-[80px] gradient-border "
            >
              Become a member
            </button>
          </a>
        </div>
      </div>
    </div>
  </section>
  <section
    class="relative  xl:px-0 px-6 md:py-32 py-16 undefined"
  >
    <img
      alt="gradient overlay"
      class="hidden md:block absolute bg-center top-0 left-0  right-0 w-full h-full z-[-10]"
      src="/images/gradient-overlay.png"
    />
    <img
      alt="gradient overlay mobile"
      class="block md:hidden absolute bg-center top-0 left-0  right-0 w-full h-full z-[-10]"
      src="/images/gradient-overlay-mobile.png"
    />
    <div
      class="max-w-[832px] w-full mx-auto md:py-0 py-8 flex justify-center items-center flex-col text-center md:bg-transparent bg-tones-white rounded-3xl md:backdrop-blur-0 backdrop-blur-12 "
    >
      <img
        aria-label="message icon"
        class="!mb-0"
        src="/images/icons/message.svg"
      />
      <p
        class="text-white text-4xl md:text-5xl leading-[122.222%] md:leading-[116.667%] font-normal mt-8 mb-6"
      >
        Connect with the community
      </p>
      <span
        class=" text-lightgrey tab-button-text"
      >
        Join our Slack channel to discuss work and reach out to project maintainers.
      </span>
      <a
        href="/en/slack"
      >
        <button
          class="rounded-2xl bg-transparent gradient-border border-2 border-pink-500/0 text-white text-base leading-6 font-normal w-[179px] h-[48px]  transition-all duration-500 ease-in-out  hidden md:block mt-8"
        >
          Learn More
        </button>
      </a>
    </div>
  </section>
  <section
    class="relative bg-gradient-to-br from-[#0E002A] via-[#1A0B3D] to-[#0E002A] overflow-hidden"
  >
    <div
      class="absolute inset-0 opacity-5"
    >
      <div
        class="absolute inset-0 bg-repeat"
      />
    </div>
    <div
      class="relative max-w-7xl px-6 py-16 mx-auto text-center lg:py-24"
    >
      <div
        class="relative max-w-5xl mx-auto"
      >
        <div
          class="bg-white/5 backdrop-blur-sm rounded-3xl border border-white/10 p-8 md:p-12 shadow-2xl"
        >
          <figure>
            <div
              class="mb-8"
            >
              <svg
                class="h-12 w-12 mx-auto text-pink opacity-60"
                fill="none"
                viewBox="0 0 24 27"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M14.017 18L14.017 10.609C14.017 4.905 17.748 1.039 23 0L23.995 2.151C21.563 3.068 20 5.789 20 8H24V18H14.017ZM0 18V10.609C0 4.905 3.748 1.038 9 0L9.996 2.151C7.563 3.068 6 5.789 6 8H9.983L9.983 18L0 18Z"
                  fill="currentColor"
                />
              </svg>
            </div>
            <blockquote
              class="slide-in-right mb-8"
            >
              <div
                class="min-h-[200px] md:min-h-[180px] flex items-center justify-center"
              >
                <p
                  class="text-xl md:text-2xl leading-relaxed font-medium text-white/90 italic"
                >
                  "
                  Microsoft is proud to be a Strategic Member of the Eclipse Adoptium Working Group, collaborating closely with industry partners to contribute to and benefit from a vibrant open-source ecosystem. Together, we are dedicated to delivering high-quality, secure, and performant Java runtimes that empower our customers and the developer community.
                  "
                </p>
              </div>
            </blockquote>
            <figcaption
              class="flex flex-col sm:flex-row items-center justify-center gap-4"
            >
              <img
                alt="Martijn Verburg"
                class="w-16 h-16 rounded-full border-2 border-pink-400/30 object-cover"
                src="/images/authors/martijnverburg.jpg"
              />
              <div
                class="text-center sm:text-left"
              >
                <div
                  class="text-lg font-semibold text-white"
                >
                  Martijn Verburg
                </div>
                <div
                  class="text-pink font-medium"
                >
                  Principal Group Manager - Java & Go @ Microsoft
                </div>
              </div>
            </figcaption>
          </figure>
        </div>
      </div>
      <div
        class="flex justify-center items-center mt-12 space-x-6"
      >
        <button
          aria-label="Previous Testimonial"
          class="group p-3 rounded-full bg-white/10 border border-white/20 hover:bg-white/20 hover:border-pink-400/50 transition-all duration-300"
        >
          <svg
            class="w-5 h-5 text-white group-hover:text-pink transition-colors"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 448 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M9.4 233.4c-12.5 12.5-12.5 32.8 0 45.3l160 160c12.5 12.5 32.8 12.5 45.3 0s12.5-32.8 0-45.3L109.2 288 416 288c17.7 0 32-14.3 32-32s-14.3-32-32-32l-306.7 0L214.6 118.6c12.5-12.5 12.5-32.8 0-45.3s-32.8-12.5-45.3 0l-160 160z"
            />
          </svg>
        </button>
        <div
          class="flex space-x-2"
        >
          <button
            aria-label="Go to testimonial 1"
            class="h-2 rounded-full transition-all duration-300 w-8 bg-pink-400"
          />
          <button
            aria-label="Go to testimonial 2"
            class="h-2 rounded-full transition-all duration-300 w-2 bg-white/30 hover:bg-white/50"
          />
          <button
            aria-label="Go to testimonial 3"
            class="h-2 rounded-full transition-all duration-300 w-2 bg-white/30 hover:bg-white/50"
          />
        </div>
        <button
          aria-label="Next Testimonial"
          class="group relative p-3 rounded-full bg-white/10 border border-white/20 hover:bg-white/20 hover:border-pink-400/50 transition-all duration-300 overflow-hidden"
        >
          <svg
            class="w-5 h-5 text-white group-hover:text-pink transition-colors relative z-10"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 448 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M438.6 278.6c12.5-12.5 12.5-32.8 0-45.3l-160-160c-12.5-12.5-32.8-12.5-45.3 0s-12.5 32.8 0 45.3L338.8 224 32 224c-17.7 0-32 14.3-32 32s14.3 32 32 32l306.7 0L233.4 393.4c-12.5 12.5-12.5 32.8 0 45.3s32.8 12.5 45.3 0l160-160z"
            />
          </svg>
          <div
            class="absolute inset-0 rounded-full border-2 border-transparent border-t-pink-400 animate-spin opacity-60"
            style="animation-duration: 8s;"
          />
        </button>
      </div>
      <div
        class="flex justify-center items-center mt-12 space-x-8"
      >
        <button
          aria-label="Microsoft Testimonial"
          class="transition-all duration-300 opacity-60 hover:opacity-100 opacity-100 scale-110"
        >
          <div
            class="h-12 w-32 flex items-center justify-center"
          >
            <img
              alt="Microsoft"
              class="max-h-full max-w-full object-contain brightness-0 invert"
              src="/images/microsoft.svg"
            />
          </div>
        </button>
        <button
          aria-label="Alibaba Cloud Testimonial"
          class="transition-all duration-300 opacity-60 hover:opacity-100 hover:scale-105"
        >
          <div
            class="h-12 w-32 flex items-center justify-center"
          >
            <img
              alt="Alibaba Cloud"
              class="max-h-full max-w-full object-contain brightness-0 invert"
              src="/images/alibaba.png"
            />
          </div>
        </button>
        <button
          aria-label="Azul Systems Testimonial"
          class="transition-all duration-300 opacity-60 hover:opacity-100 hover:scale-105"
        >
          <div
            class="h-12 w-32 flex items-center justify-center"
          >
            <img
              alt="Azul Systems"
              class="max-h-full max-w-full object-contain brightness-0 invert"
              src="/images/azul.svg"
            />
          </div>
        </button>
      </div>
    </div>
  </section>
  <div
    class="bg-purple py-16 lg:pt-32 pb-16 xl:px-0 lg:px-8 px-0"
  >
    <div
      class="mx-auto max-w-[1264px] w-full flex lg:flex-row flex-col items-start lg:items-center justify-center lg:space-x-8 xl:space-x-16 relative overflow-hidden"
    >
      <div
        class="max-w-[436px] w-full mb-16 lg:mb-0 px-6"
      >
        <h2
          class="here text-4xl lg:text-5xl leading-[44px] lg:leading-[56px] font-semibold text-white-900"
        >
          Latest news and updates
        </h2>
        <h3
          class="text-xl font-normal leading-7 text-grey mt-6 mb-8"
        >
          Follow the latest updates from the Eclipse Adoptium Project
        </h3>
        <a
          href="/en/news"
        >
          <button
            class="rounded-2xl bg-transparent gradient-border border-2 border-pink-500/0 text-white text-base leading-6 font-bold w-[154px] h-[48px] block"
          >
            See all news
          </button>
        </a>
      </div>
      <div
        class="max-w-[780px] w-full lg:flex gap-4 xl:gap-x-8 hidden"
      >
        <div
          class="max-w-[374px] w-full lg:flex hidden flex-col space-y-4 xl:space-y-8"
        >
          <div
            class="bg-white/5 rounded-3xl border border-white/50 backdrop-blur-xl p-8 xl:p-10"
          >
            <h2
              class="text-primary text-lg leading-6 font-bold m-0 flex items-center gap-x-3"
            >
              <svg
                fill="none"
                height="14"
                viewBox="0 0 14 14"
                width="14"
                xmlns="http://www.w3.org/2000/svg"
              >
                <rect
                  fill="#FF1464"
                  height="8"
                  rx="2"
                  width="8"
                  x="3"
                  y="3"
                />
                <rect
                  height="11"
                  rx="3.5"
                  stroke="#FF1464"
                  stroke-opacity="0.25"
                  stroke-width="3"
                  width="11"
                  x="1.5"
                  y="1.5"
                />
              </svg>
              News
            </h2>
            <p
              class="text-white text-lg xl:text-xl font-normal leading-6 xl:leading-7 my-2"
            >
              Mock Title 1
            </p>
            <span
              class="text-sm text-grey font-normal leading-5 block"
            >
              2021-01-01
            </span>
            <a
              class="py-3 text-base underline font-bold leading-6 text-white mt-2 block border-white w-fit"
            >
              Read More
            </a>
          </div>
          <div
            class="bg-white/5 rounded-3xl border border-white/50 backdrop-blur-xl p-8 xl:p-10"
          >
            <h2
              class="text-primary text-lg leading-6 font-bold m-0 flex items-center gap-x-3"
            >
              <svg
                fill="none"
                height="14"
                viewBox="0 0 14 14"
                width="14"
                xmlns="http://www.w3.org/2000/svg"
              >
                <rect
                  fill="#FF1464"
                  height="8"
                  rx="2"
                  width="8"
                  x="3"
                  y="3"
                />
                <rect
                  height="11"
                  rx="3.5"
                  stroke="#FF1464"
                  stroke-opacity="0.25"
                  stroke-width="3"
                  width="11"
                  x="1.5"
                  y="1.5"
                />
              </svg>
              News
            </h2>
            <p
              class="text-white text-lg xl:text-xl font-normal leading-6 xl:leading-7 my-2"
            >
              Mock Title 2
            </p>
            <span
              class="text-sm text-grey font-normal leading-5 block"
            >
              2021-01-01
            </span>
            <a
              class="py-3 text-base underline font-bold leading-6 text-white mt-2 block border-white w-fit"
            >
              Read More
            </a>
          </div>
        </div>
        <div
          class="max-w-[374px] w-full lg:flex hidden flex-col space-y-4 xl:space-y-8 mt-16"
        >
          <div
            class="bg-white/5 rounded-3xl border border-white/50 backdrop-blur-xl p-8 xl:p-10"
          >
            <h2
              class="text-primary text-lg leading-6 font-bold m-0 flex items-center gap-x-3"
            >
              <svg
                fill="none"
                height="14"
                viewBox="0 0 14 14"
                width="14"
                xmlns="http://www.w3.org/2000/svg"
              >
                <rect
                  fill="#FF1464"
                  height="8"
                  rx="2"
                  width="8"
                  x="3"
                  y="3"
                />
                <rect
                  height="11"
                  rx="3.5"
                  stroke="#FF1464"
                  stroke-opacity="0.25"
                  stroke-width="3"
                  width="11"
                  x="1.5"
                  y="1.5"
                />
              </svg>
              News
            </h2>
            <p
              class="text-white text-lg xl:text-xl font-normal leading-6 xl:leading-7 my-2"
            >
              Mock Title 3
            </p>
            <span
              class="text-sm text-grey font-normal leading-5 block"
            >
              2021-01-01
            </span>
            <a
              class="py-3 text-base underline font-bold leading-6 text-white mt-2 block border-white w-fit"
            >
              Read More
            </a>
          </div>
          <div
            class="bg-white/5 rounded-3xl border border-white/50 backdrop-blur-xl p-8 xl:p-10"
          >
            <h2
              class="text-primary text-lg leading-6 font-bold m-0 flex items-center gap-x-3"
            >
              <svg
                fill="none"
                height="14"
                viewBox="0 0 14 14"
                width="14"
                xmlns="http://www.w3.org/2000/svg"
              >
                <rect
                  fill="#FF1464"
                  height="8"
                  rx="2"
                  width="8"
                  x="3"
                  y="3"
                />
                <rect
                  height="11"
                  rx="3.5"
                  stroke="#FF1464"
                  stroke-opacity="0.25"
                  stroke-width="3"
                  width="11"
                  x="1.5"
                  y="1.5"
                />
              </svg>
              News
            </h2>
            <p
              class="text-white text-lg xl:text-xl font-normal leading-6 xl:leading-7 my-2"
            >
              Mock Title 4
            </p>
            <span
              class="text-sm text-grey font-normal leading-5 block"
            >
              2021-01-01
            </span>
            <a
              class="py-3 text-base underline font-bold leading-6 text-white mt-2 block border-white w-fit"
            >
              Read More
            </a>
          </div>
        </div>
      </div>
      <div
        class="w-full flex relative lg:hidden"
      >
        <div
          class="overflow-hidden pl-6 max-w-full relative"
        >
          <div
            data-testid="Swiper"
          >
            <div
              data-testid="SwiperSlide"
            >
              <h2
                class="text-primary text-base leading-6 font-bold m-0 flex items-center gap-x-3"
              >
                <svg
                  fill="none"
                  height="14"
                  viewBox="0 0 14 14"
                  width="14"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <rect
                    fill="#FF1464"
                    height="8"
                    rx="2"
                    width="8"
                    x="3"
                    y="3"
                  />
                  <rect
                    height="11"
                    rx="3.5"
                    stroke="#FF1464"
                    stroke-opacity="0.25"
                    stroke-width="3"
                    width="11"
                    x="1.5"
                    y="1.5"
                  />
                </svg>
                News
              </h2>
              <p
                class="text-white text-xl  font-normal leading-7 mt-6"
              >
                Mock Title 1
              </p>
              <span
                class="text-sm text-grey font-normal leading-5 block mt-2 mb-6"
              >
                2021-01-01
              </span>
              <a
                class="py-3 text-base underline font-bold leading-6 text-white mt-2 block border-white w-fit"
              >
                Read More
              </a>
            </div>
            <div
              data-testid="SwiperSlide"
            >
              <h2
                class="text-primary text-base leading-6 font-bold m-0 flex items-center gap-x-3"
              >
                <svg
                  fill="none"
                  height="14"
                  viewBox="0 0 14 14"
                  width="14"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <rect
                    fill="#FF1464"
                    height="8"
                    rx="2"
                    width="8"
                    x="3"
                    y="3"
                  />
                  <rect
                    height="11"
                    rx="3.5"
                    stroke="#FF1464"
                    stroke-opacity="0.25"
                    stroke-width="3"
                    width="11"
                    x="1.5"
                    y="1.5"
                  />
                </svg>
                News
              </h2>
              <p
                class="text-white text-xl  font-normal leading-7 mt-6"
              >
                Mock Title 2
              </p>
              <span
                class="text-sm text-grey font-normal leading-5 block mt-2 mb-6"
              >
                2021-01-01
              </span>
              <a
                class="py-3 text-base underline font-bold leading-6 text-white mt-2 block border-white w-fit"
              >
                Read More
              </a>
            </div>
            <div
              data-testid="SwiperSlide"
            >
              <h2
                class="text-primary text-base leading-6 font-bold m-0 flex items-center gap-x-3"
              >
                <svg
                  fill="none"
                  height="14"
                  viewBox="0 0 14 14"
                  width="14"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <rect
                    fill="#FF1464"
                    height="8"
                    rx="2"
                    width="8"
                    x="3"
                    y="3"
                  />
                  <rect
                    height="11"
                    rx="3.5"
                    stroke="#FF1464"
                    stroke-opacity="0.25"
                    stroke-width="3"
                    width="11"
                    x="1.5"
                    y="1.5"
                  />
                </svg>
                News
              </h2>
              <p
                class="text-white text-xl  font-normal leading-7 mt-6"
              >
                Mock Title 3
              </p>
              <span
                class="text-sm text-grey font-normal leading-5 block mt-2 mb-6"
              >
                2021-01-01
              </span>
              <a
                class="py-3 text-base underline font-bold leading-6 text-white mt-2 block border-white w-fit"
              >
                Read More
              </a>
            </div>
            <div
              data-testid="SwiperSlide"
            >
              <h2
                class="text-primary text-base leading-6 font-bold m-0 flex items-center gap-x-3"
              >
                <svg
                  fill="none"
                  height="14"
                  viewBox="0 0 14 14"
                  width="14"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <rect
                    fill="#FF1464"
                    height="8"
                    rx="2"
                    width="8"
                    x="3"
                    y="3"
                  />
                  <rect
                    height="11"
                    rx="3.5"
                    stroke="#FF1464"
                    stroke-opacity="0.25"
                    stroke-width="3"
                    width="11"
                    x="1.5"
                    y="1.5"
                  />
                </svg>
                News
              </h2>
              <p
                class="text-white text-xl  font-normal leading-7 mt-6"
              >
                Mock Title 4
              </p>
              <span
                class="text-sm text-grey font-normal leading-5 block mt-2 mb-6"
              >
                2021-01-01
              </span>
              <a
                class="py-3 text-base underline font-bold leading-6 text-white mt-2 block border-white w-fit"
              >
                Read More
              </a>
            </div>
          </div>
          <div
            class="flex gap-4 items-center justify-center mt-16 mb-8"
          >
            <button
              aria-label="Previous News"
              class="text-white"
            >
              <svg
                fill="none"
                height="50"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="1"
                viewBox="0 0 24 24"
                width="50"
                xmlns="http://www.w3.org/2000/svg"
              >
                <circle
                  cx="12"
                  cy="12"
                  r="10"
                />
                <polyline
                  points="12 8 8 12 12 16"
                />
                <line
                  x1="16"
                  x2="8"
                  y1="12"
                  y2="12"
                />
              </svg>
            </button>
            <button
              aria-label="Next News"
              class="text-white"
            >
              <svg
                fill="none"
                height="50"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="1"
                viewBox="0 0 24 24"
                width="50"
                xmlns="http://www.w3.org/2000/svg"
              >
                <circle
                  cx="12"
                  cy="12"
                  r="10"
                />
                <polyline
                  points="12 16 16 12 12 8"
                />
                <line
                  x1="8"
                  x2="16"
                  y1="12"
                  y2="12"
                />
              </svg>
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</main>
`;
